scripts/src/answers_jsonlines_transformer.ts

```diff
@@ -0,0 +1,157 @@
+/*
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/*
+Usage: Convert answers with multiple parts into separate rows.
+
+  node build/server/setup/answers_jsonlines_transformer.js \
+    --infile="./tmp/Answers.json" \
+    --infile="./tmp/Answers2.json"
+*/
+
+import * as yargs from 'yargs';
+import * as fs from 'fs';
+import * as readline from 'readline';
+
+// Command line arguments.
+interface Params {
+  infile: string;
+  outfile: string;
+}
+
+interface AnswerT1 {
+  answer: string;
+  answer_id?: string;
+  client_job_key: string;
+  question_group_id: string;
+  question_id: string;
+  timestamp: string;
+  answer_score?: string;
+  worker_nonce: string;
+}
+
+type AnswerPartType =
+  | 'insult'
```

Comment:
Does this compile with the leading '|'? I haven't seen that before. Seems like a linter tool error?

Reply:
yeah, google's auto-formatted did this.